### PR TITLE
Azure devops: test parameters update fix

### PR DESCRIPTION
**Pull request by Brrovko**

### CODE REVIEW

File: GherkinSyncTool.Synchronizers.AzureDevOps/AzureDevopsSynchronizer.cs [hunk 1]
```diff
@@ -237,13 +237,14 @@ private IEnumerable<WitBatchRequest> GetTestCasesToUpdate(Dictionary<int, WitBat
                 {
                     return false;
                 }
-                var fieldANormalized = Regex.Replace(fieldValue, @"\s", "").ToLowerInvariant();
-                var fieldBNormalized = Regex.Replace(dictionaryB[fieldKey], @"\s", "").ToLowerInvariant();
+
+                var fieldANormalized = Regex.Replace(fieldValue ?? string.Empty, @"\s", "").ToLowerInvariant();
+                var fieldBNormalized = Regex.Replace(dictionaryB[fieldKey] ?? string.Empty, @"\s", "").ToLowerInvariant();
```
Comment (vladimir-s-quantory) [inline GherkinSyncTool.Synchronizers.AzureDevOps/AzureDevopsSynchronizer.cs:242]: If whitespace replacement logic is the same for both fields maybe we should create a single method for them both and not double this two calls?

**Review by ILastovka — APPROVED** [review]

File: GherkinSyncTool.Synchronizers.AzureDevOps/AzureDevopsSynchronizer.cs [hunk 2]
```diff
@@ -237,13 +237,14 @@ private IEnumerable<WitBatchRequest> GetTestCasesToUpdate(Dictionary<int, WitBat
...
 
                 //Compare Tags
                 if (string.Equals(fieldKey, WorkItemFields.Tags))
                 {
-                    var separator = ";";
+                    const string separator = ";";
```
Comment (vladimir-s-quantory) [inline GherkinSyncTool.Synchronizers.AzureDevOps/AzureDevopsSynchronizer.cs:247]: Can we use `char` insted of `string`?

File: GherkinSyncTool.Synchronizers.AzureDevOps/AzureDevopsSynchronizer.cs (hunk 1 above)
Comment (Brrovko) [reply GherkinSyncTool.Synchronizers.AzureDevOps/AzureDevopsSynchronizer.cs]: Done

File: GherkinSyncTool.Synchronizers.AzureDevOps/AzureDevopsSynchronizer.cs (hunk 2 above)
Comment (Brrovko) [reply GherkinSyncTool.Synchronizers.AzureDevOps/AzureDevopsSynchronizer.cs]: Done

**Review by vladimir-s-quantory — APPROVED** [review]

**Pull request merged**